@here/harp-examples/src/threejs_raycast.ts

```diff
@@ -112,18 +112,18 @@ export namespace ThreejsRaycast {
         return map;
     }
 
-    document.body.innerHTML +=
+    const template = document.createElement("template");
+    template.innerHTML =
         `<style>
-            #mapCanvas{
-                top:0;
-            }
             #info{
-                color: #000;
+                position: absolute;
+                left: 0;
```

Comment:
Why not stretch it to 100% like in other examples instead of specifying left/right properties and margin?